docs/command/commands.md

```diff
@@ -0,0 +1,84 @@
+---
+id: commands
+title: C3PM commands
+---
+### How to use c3pm commands?
+
+`ctpm add`
```

Comment:
maybe `ctpm add <package>` to explicit the fact that package (defined below) is a parameter ?